kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -2195,6 +2195,9 @@ protected void handleAddPartitionsToTxn(KafkaHeaderAndRequest kafkaHeaderAndRequ
                                             responseData.results().add(topicResult);
                                             topicPartitionErrorsMap.forEach((TopicPartition tp, Errors error) -> {
                                                 if (tp.topic().equals(topicName)) {
+                                                    if (log.isDebugEnabled() && error != Errors.NONE) {
+                                                        log.info("Error {} for {}", error, tp);
+                                                    }
```

Comment:
Why did you print info level logs when `isDebugEnabled()`?  Even if it's debug level, the log here is meaningless because we can know the logs from the client side. Kafka never prints such types of logs at the broker side if the error can be known from the client side. So I suggest removing these lines.